presto-docs/src/main/sphinx/prestissimo/prestissimo-features.rst

```diff
@@ -184,3 +184,170 @@ for cleanup. Only applicable when ``enable-old-task-cleanup`` is ``true``.
 Old task is defined as a PrestoTask which has not received heartbeat for at least
 ``old-task-cleanup-ms``, or is not running and has an end time more than
 ``old-task-cleanup-ms`` ago.
+
+
+Session Properties
+------------------
+
+Defines all system session properties supported by native worker to ensure
+that they are the source of truth and to differentiate them from Java based
+session properties.
+
+The following are the native session properties kept as the source of truth in prestissimo.
+
+``driver_cpu_time_slice_limit_ms``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``1000``
+
+Native Execution only. The cpu time slice limit in ms that a driver thread.
+If not ``zero``, can continuously run without yielding.
```

Comment:
If driver_cpu_time_slice_limit_ms is not zero, the worker can...

What if this is negative?

Reply:
upated and mentioned about the negative value case